src/meta/processors/parts/AlterSpaceProcessor.cpp

```diff
@@ -25,7 +25,18 @@ void AlterSpaceProcessor::process(const cpp2::AlterSpaceReq& req) {
     default:
       break;
   }
-  handleErrorCode(nebula::cpp2::ErrorCode::SUCCEEDED);
+  std::vector<kvstore::KV> data;
```

Comment:
Write all data in a single batch (both data in `addZones` and update time), you could see other processors.